tests/Feature/Proxy/RestartProxyTest.php

```
@@ -0,0 +1,139 @@
+        Queue::assertPushed(RestartProxyJob::class, 2);
+
+        // Get the jobs
+        $jobs = Queue::pushed(RestartProxyJob::class);
+
+        // Verify both jobs have WithoutOverlapping middleware
+        foreach ($jobs as $job) {
+            $middleware = $job['job']->middleware();
+            $this->assertCount(1, $middleware);
+            $this->assertInstanceOf(\Illuminate\Queue\Middleware\WithoutOverlapping::class, $middleware[0]);
+        }
+    }
```

PHPMD (2.15.0)

The method test_restart_prevents_concurrent_jobs_via_without_overlapping is not named in camelCase. (undefined)

(CamelCaseMethodName)